CONTRIBUTING.md

```diff
@@ -0,0 +1,93 @@
+# How to contribute
+
+oria-operator is Apache 2.0 licensed and accepts contributions via GitHub pull requests. This document outlines some of the conventions on commit message formatting, contact points for developers, and other resources to help get contributions into oria-operator.
```

Comment:
`oria-operator` is Apache 2.0 licensed and accepts contributions via GitHub pull requests. This document outlines some of the conventions on commit message formatting, contact points for developers, and other resources to help get contributions into `oria-operator`.

Reply:
I think this is resolved with commit https://github.com/operator-framework/oria-operator/commit/c387d7632b2f440788ea0a5cb39293fb9ddbbea2